src/main/java/org/jitsi/srtp/crypto/Aes.java

```diff
@@ -295,18 +302,24 @@ private static CipherFactory benchmark(
                 }
                 else
                 {
+                    /* If this is the user-chosen factory class, we don't really
+                     * need to benchmark it; just verify that it's working.
+                     */
+                    int numWarmups = chosenFactoryClass ? 0 : NUM_WARMUPS;
+                    int numBenchmarks = chosenFactoryClass ? 1 : NUM_BENCHMARKS;
+
                     BenchmarkOperation benchmark = BenchmarkOperation.getBenchmark(transformation, keySize);
-                    if (warmup)
+                    if (!chosenFactoryClass)
                     {
                         // Let the JVM "warm up" (do JIT compilation and the like)
-                        for (int i = 0; i < NUM_WARMUPS; i++)
+                        for (int i = 0; i < numWarmups; i++)
                         {
                             benchmark.run(cipher);
                         }
                     }
 
                     long startTime = System.nanoTime();
-                    for (int i = 0; i < NUM_BENCHMARKS; i++)
+                    for (int i = 0; i < numBenchmarks; i++)
```

Comment:
Should line 329 use `numBenchmarks` instead of `NUM_BENCHMARKS`? I guess in practice it doesn't matter